tests/positive/files/file.go

```diff
@@ -65,6 +65,40 @@ func CreateFileOnRoot() types.Test {
 	}
 }
 
+func CreateFileOnRootFromBase64() types.Test {
```

Comment:
This needs to be registered at the top of the file.

Reply:
Thanks!